assets_manager/models.py

```diff
@@ -58,3 +61,8 @@ def get_full_absolute_url(self):
         domain = Site.objects.get_current().domain
 
         return "http://{0}{1}".format(domain, self.get_absolute_url)
+
+
+@receiver(pre_delete, sender=Asset)
+def asset_delete(sender, instance, **kwargs):
+    instance.file.delete(False)
```

Comment:
isso remove o arquivo do filesystem? pelo que diz nesta resporta é um pouco mais trabalhoso (http://stackoverflow.com/a/33081018/1503)

E melhor verificar se a instância tem file... e loggar erro de IO se for o caso

```python
try: 
    if instance.file:
        instance.file.delete()
except IOError:
    logger.error('Erro deletando o asset (pk: %s)', instance.pk)
```